lib/util/QueuePool.js

```diff
@@ -24,8 +24,12 @@ class QueuePool {
             this.queues.push(new Queue(fn));
         }
         this.overflow = new Queue(this._overflow.bind(this));
+        this._overflowCount = 0;
     }
 
+    get waitingItems() { return this._overflowCount; };
```

Comment:
This is unused within the context of this commit, it's generally clearer to include usage of a thing with its creation within the same commit. (But don't worry about it this time around).